v1/examples/rp-authz-request.json

```diff
@@ -0,0 +1,10 @@
+{
+  "claims": "%7B%22userinfo%22%3A%7B%22https%3A%2F%2Fvocab.account.gov.uk%2Fv1%2FcoreIdentityJWT%22%3A%7B%22essential%22%3Atrue%7D%7D%7D",
+  "response_type": "code",
+  "redirect_uri": "http://localhost:3001/oauth/callback",
+  "state": "tRNlWtowYYcSBUzFfkjXbn8GX0V4RansaA2USbjZmlQ",
+  "client_id": "IcVM2g9s10dXL0Jpd4_rVfQ_G0Y",
+  "scope": "openid email phone offline_access",
```

Comment:
I don't think we do `offline_access` but it might be worth pinging @mrwilson to see if there is a sample/test fixture RP Authz request we can/should be keeping this in sync with.

Reply:
This example I think was taken from the RPStub that @zapthedingbat had put together and it is contained in some documentation that I have seen from @philf999, but I will check with @mrwilson.

Reply:
Hmmmmm okay, does that work for identity proofing data? The VC from SPOT should expire pretty rapidly after going through so this seems somewhat dubious to me.

Reply:
No, it doesn't we can only pull the SPOT output (on a call to /userinfo) once, then we delete after first read. This is more for ID tokens than actual identity info.

Reply:
We should probably make this clear in the docs if it isn't already - for me that would be reason enough to decide *not* to support it, but maybe there are use cases I haven't thought about. Happy to resolve this convo here but I'll cc @zapthedingbat as this covers RP happiness :-)

Reply:
Also cc @philf999

Reply:
Myself and @zapthedingbat  have wondered what the refresh token would be used for. As mentioned the claims that are retrieved from userinfo expire as quickly as the access_token. I'm not sure if there is any use case for refreshing an id_token.

Reply:
There's ongoing discussion in the context of VC storage where one of the solutions involves access tokens issued from Orchestration to IPV Core (this is in no way the settled solution for this, fwiw)